react-native/react/tracker/index.js

```diff
@@ -69,6 +71,7 @@ class Tracker extends Component {
         onFollowChecked: this.props.onFollowChecked
       },
       proofsProps: {
+        username: this.props.username,
```

Comment:
so theoretically this index. could be shared between RN and electron but we're leaking info about the rendering, like which panes have which properties and whatnot. That's kinda why I advocated doing the splitting in render.desktop.js. When we do merge this and mobile let's revisit and see if it's where it belongs